test/database.ts

```diff
@@ -573,6 +574,94 @@ describe('Database', () => {
     });
   });
 
+  describe('batchWrite', () => {
```

Comment:
Add a test case for "mutationGroups" passed as empty list or null.

Reply:
Done